src/testInteg/codecatalyst/devEnv.test.ts

```diff
@@ -0,0 +1,236 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as assert from 'assert'
+import { InactivityMessage, shouldTrackUserActivity } from '../../codecatalyst/devEnv'
```

Comment:
Looks like the devfileLocation logic should also live in this new devEnv.ts file. And its tests should be moved here. 

https://github.com/aws/aws-toolkit-vscode/blob/bd58f9e5e48bc109827d019442c2cdf97bc0afb6/src/test/codecatalyst/devfileLocation.test.ts#L13

Reply:
I see we also have a `devfile.ts` file as well. Will look later to see how to refactor these properly.